summary/README.md

```diff
@@ -0,0 +1,780 @@
+# Automated Test Case Generation
+
+
+## Introduction
+To automatically generate test cases, there are two things we needs to do: 
+- Search through the problem space to generate a random population of test cases.
+- Apply a genetic algorithm to evolve the randomly generated test case population according to a fitness function.
+
+There are different approaches to which this method can be applied: single test generation (target one goal at a time), or whole test suite generation.  
+After executing our own literature survey of six papers treating the matter, we have received a clear overview of what search algorithms  [[1](#1)] exist to find a suitable test case population. 
```

Comment:
Is the term `received` correct here? It could be that I've just never used it in this way. I'm asking because we didn't receive an overview, but we created it.
*There's a double space before the citation though*